includes/admin.form.inc

```diff
@@ -94,5 +94,78 @@ function islandora_basic_collection_admin(array $form, array &$form_state) {
       ),
     ),
   );
+
+  // Define the elements that appear on a collection objects display page.
+  // The key's match up with the form elements array keys.
+  $page_content = array(
+    'description' => array(
+      'name' => "Description",
+      'description' => "An objects description field",
+    ),
+    'collections' => array(
+      'name' => "In Collection",
+      'description' => "Indicates which collections this object belongs to",
+    ),
+    'wrapper' => array(
+      'name' => "Fieldset Metadata",
+      'description' => "An objects metadata collection set",
+    ),
+    'islandora_basic_collection_display' => array(
+      'name' => "Object Content",
+      'description' => "Main object page content, such as configured viewers",
```

Comment:
The inner human-readable values here (throughout this array) should probably be `t()`-wrapped.